samples/data-grid/basic/cell-format/demo.html

```diff
@@ -0,0 +1,12 @@
+<script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+
+<div id="container"></div>
+<p class="highcharts-description">
+    A simple DataGrid with a custom cell format. Some of the changes:
```

Comment:
Aquestion of personal taste, but I'd prefer a `padding-left` in the description. Currently "A simple ..." is almost on the border.